webhook/psbinding/psbinding.go

```diff
@@ -297,14 +306,20 @@ func (ac *Reconciler) reconcileMutatingWebhook(ctx context.Context, caCert []byt
 	// This is only supported by 1.15+ clusters.
 	matchPolicy := admissionregistrationv1beta1.Equivalent
 
+	// See if the opt-out behaviour has been specified and specify the Inclusion Selector.
+	selector := ExclusionSelector
+	if HasOptOutBindingSelector(ctx) {
```

Comment:
Is the idea that to opt-out, you would use `WithContext` to infuse the context with some state from a configmap?

I was just thinking that we'd make the selector a field on Reconciler 🤔